<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convex Hull</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="Draw" name="Canvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Amount of points:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>750000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pattern:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gen_pattern">
       <item>
        <property name="text">
         <string>Random</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Grid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Elipse</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generate">
       <property name="text">
        <string>Generate points</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Jarvis Scan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quick Hull</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sweep Line</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Create convex hull</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Duration in [s]:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time_c">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Clear points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Clear convex hull</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Draw</class>
   <extends>QWidget</extends>
   <header>draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
